pkg/virt-launcher/virtwrap/network/network_test.go

```diff
@@ -45,6 +53,7 @@ var _ = Describe("VMNetworkConfigurator", func() {
 				network:          defaultNet,
 				handler:          vmNetworkConfigurator.handler,
 				cacheFactory:     vmNetworkConfigurator.cacheFactory,
+				dhcpConfigurator: &dhcpConfigurator,
```

Comment:
Since no change was done to `network.go` this test change is very suspicious. I think `newPodNIC` should be moved to `network.go` or tested as part of `podInterface.go` and not here.

Reply:
It can be done in a follow-up. Since we have a lot of follow-up promised. Is there some list I can track with all the follow-up?

Reply:
This very same PR's description.

Thanks for reminding me of this one; it would have slipped by.